tests/unit/components/question-editor/QuestionEditor.spec.jsx

```diff
@@ -1,33 +1,43 @@
 import QuestionEditor from '~/components/question-editor/QuestionEditor'
 import { screen, fireEvent } from '@testing-library/react'
 import { renderWithProviders } from '~tests/test-utils'
-import { beforeEach, describe } from 'vitest'
-
-const data = {
-  type: 'openAnswer',
-  title: 'question 1',
-  category: null,
-  text: '',
-  openAnswer: '',
-  answers: []
-}
+import { afterEach, beforeEach, describe, expect } from 'vitest'
+
 const handleInputChange = vi.fn()
 const handleNonInputValueChange = vi.fn()
 const onEditMock = vi.fn()
+const onSave = vi.fn()
 
 const props = {
-  data,
   handleInputChange,
   handleNonInputValueChange,
   onCancel: vi.fn(),
   onEdit: onEditMock,
-  onSave: vi.fn(),
+  onSave,
   isQuizQuestion: true
 }
 
-describe('QuestionEditor component', () => {
+describe('QuestionEditor component with an open question type', () => {
+  const openQuestionData = {
+    type: 'openAnswer',
+    title: 'question 1',
+    category: null,
+    text: 'question text',
+    openAnswer: 'answer',
+    answers: []
+  }
+
+  const openQuestionProps = {
+    ...props,
+    data: openQuestionData
+  }
```

Comment:
Maybe it would be better to move these constants outside of `describe` block? Because `props` are outside it, but `openQuestionData` and `openQuestionProps` are not

Reply:
Okay, got it. I didn't notice, that those constants are used only within one describe block. If so, it is good to go.